include/Mixer.h

```diff
@@ -88,11 +89,20 @@ class MixerChannel : public ThreadableJob
 		std::atomic_size_t m_dependenciesMet;
 		void incrementDeps();
 		void processed();
-		
+
+		auto silenced() -> bool { return m_silenced.load(std::memory_order_relaxed); }
+
+		static auto silenceInvalidOutput() -> bool
+		{
+			static auto s_value = ConfigManager::inst()->value("audioengine", "silenceinvalidmixeroutput", "0").toInt();
```

Comment:
Is this disabled by default now?  I thought we were keeping this feature enabled to avoid regressions.  My approval was based on this testing: https://github.com/LMMS/lmms/pull/7323#issuecomment-2231231247.  Since the behavior has changed, I'm re-requesting my own review as the previous review is now outdated.

Reply:
Oh yeah, I'll make it enabled by default soon.

Reply:
Just to update this, I will re-test once the "enabled by default" portion is tackled.